### PR TITLE
Added easy to use custom encoding to VelocityTemplateEngine.

### PR DESCRIPTION
**Pull request by nicosio2**
This allows the user to easily choose an encoding using the constructor.

### CODE REVIEW

**Comment by cperezs** [comment]
This would be very useful, since UTF-8 files are not rendered correctly

File: spark-template-velocity/src/main/java/spark/template/velocity/VelocityTemplateEngine.java [hunk 1]
```diff
@@ -63,7 +105,13 @@ public VelocityTemplateEngine(VelocityEngine velocityEngine) {
      */
     @Override
     public String render(ModelAndView modelAndView) {
-        Template template = velocityEngine.getTemplate(modelAndView.getViewName());
+    	Template template;
+    	if(encoding != null || !encoding.isEmpty()) {
+    		template = velocityEngine.getTemplate(modelAndView.getViewName(), encoding);
```
Comment (tipsy) [inline spark-template-velocity/src/main/java/spark/template/velocity/VelocityTemplateEngine.java]: Please use the Spark code style: https://github.com/perwendel/spark/tree/master/config

File: spark-template-velocity/src/main/java/spark/template/velocity/VelocityTemplateEngine.java [hunk 2]
```diff
@@ -45,17 +50,54 @@ public VelocityTemplateEngine() {
...
+        this.encoding = encoding;
+    }
+    
+    /**
+     * Constructor
+     *
+     * @param velocityEngine The velocity engine, must not be null.
+     */
+    public VelocityTemplateEngine(VelocityEngine velocityEngine) {
+    	this(velocityEngine, (String) null);
```
Comment (tipsy) [inline spark-template-velocity/src/main/java/spark/template/velocity/VelocityTemplateEngine.java]: String nullEncoding = null; ?

**Comment by tipsy** [comment]
Fixed

**Pull request closed without merging**